### PR TITLE
man: netdev: several cleanups

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: man/systemd.netdev.xml [hunk 1]
```diff
@@ -459,7 +459,7 @@
         <listitem>
           <para>Defines a mapping of Linux internal packet priority (<constant>SO_PRIORITY</constant>) to VLAN header
           PCP field for outgoing frames. Takes a whitespace-separated list of unsigned integer pairs in the format
-          <literal>from</literal>-<literal>to</literal>, e.g., <literal>21-7 45-5</literal> ranges 1–4294967294.
+          <literal>from</literal>-<literal>to</literal>, e.g., <literal>21-7 45-5</literal> ranges 1…4294967294.
```
Comment (keszybz) [inline man/systemd.netdev.xml]: I don't understand this sentence at all. Is "ranges" used in the sense of "is equivalent to" ?

File: man/systemd.netdev.xml [hunk 2]
```diff
@@ -470,7 +470,7 @@
         <listitem>
           <para>Defines a mapping of Linux internal packet priority (<constant>SO_PRIORITY</constant>) to VLAN header
           PCP field for incoming frames. Takes a whitespace-separated list of unsigned integer pairs in the format
-          <literal>from</literal>-<literal>to</literal>, e.g., <literal>21-7 45-5</literal> ranges 1–4294967294.
+          <literal>from</literal>-<literal>to</literal>, e.g., <literal>21-7 45-5</literal> ranges 1…4294967294.
```
Comment (keszybz) [inline man/systemd.netdev.xml]: Same here obviously.

**Review by keszybz — COMMENTED** [review]
LGTM, but I don't understand the sentence that is being touched. I think it should be fixed while at it.

**Comment by yuwata** [comment]
One commit is added to reword the sentence. PTAL.

**Comment by keszybz** [comment]
Ah, now I see that I completely misunderstood this.

**Pull request merged**